src/main/java/lotto/domian/Rank.java

```diff
@@ -0,0 +1,38 @@
+package lotto.domian;
+
+import java.util.Arrays;
+
+public enum Rank {
+
+    FIRST(6, 2000000000),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
```

자바에서는 언더바(`_`)를 통해 가독성을 높힐 수 있습니다.